content/en/tracing/troubleshooting/tracer_debug_logs.md

```diff
@@ -30,9 +30,41 @@ To enable debug mode for the Datadog Java Tracer, set the flag `-Ddd.trace.debug
 
 {{< programming-lang lang="python" >}}
 
-To enable debug mode for the Datadog Python Tracer, set the environment variables `DD_TRACE_DEBUG=true` and `DD_CALL_BASIC_CONFIG=true` when using `ddtrace-run`.
+To enable debug mode for the Datadog Python Tracer, review the instructions below for the specific scenario that applies to the application being traced. Logs emitted by the Python Tracer contain the name **[ddtrace]**.
+
+#### 1.3.0 or higher
+
```

Comment:
Why don't we mention `DD_TRACE_LOG_FILE_LEVEL` ? I actually don't totally understand why we'd have it in the first place, should we not add it in our corp docs?

Reply:
`DD_TRACE_LOG_FILE_LEVEL` doesn't need to be configured in order for logs to be written to a file. Since it wouldn't be part of the normal debug flow, I left it out of the instructions to keep it as simple as possible.

The PR https://github.com/DataDog/dd-trace-py/pull/3214 covers how `DD_TRACE_LOG_FILE_LEVEL` affects logging behavior when used with `DD_TRACE_DEBUG`. For anyone interested, it'll be in the API docs instead.

As for a use case of `DD_TRACE_LOG_FILE_LEVEL`, I think a good one would be if we want as little `DEBUG` logs as possible, but want to target exception logs more carefully, but that's still a rare scenario. I left it there as an option, but it'll be really rare to recommend it.

Reply:
Thanks for the explanation! The plan seems good!